playbooks/common/openshift-cluster/redeploy-certificates/nodes-evacuation.yml

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
Node evacuation, service restarts, etc all seem to be more generic actions that aren't necessarily tied to certificate redeployment. Should they exist outside of the redeploy-certificates dir and be included in from more generic directories?